filter_not_in_servicetype_list.go

```diff
@@ -0,0 +1,47 @@
+package aiven
+
+var _ Filter = (*NotInServiceTypeListFilter)(nil)
+
+type (
+	// List is a list of strings.
+	List map[string]struct{}
+
+	// NotInServiceTypeListFilter is a filter implementation that filters out elements that are not in the given list.
+	NotInServiceTypeListFilter struct {
+		StubFilter
```

Comment:
usage of `BasicFilter`
update: renamed `BasicFilter` to `StubFilter`, please interpret `BasicFilter` in comments as `StubFilter`

abstract: provides methods for other filters that aren't required to be reimplemented, i.e. `setResult`, `Result`, `SpreadResult` and `copy`

Reply:
this is no longer relevant